src/react_barchart.jsx

```diff
@@ -0,0 +1,237 @@
+import ReactDOM from 'react-dom'
```

Comment:
Your library should not use ReactDOM directly, it would only be used in `demo.jsx`